example/advection/advection_package.cpp

```diff
@@ -104,19 +109,18 @@ AmrTag CheckRefinement(Container<Real> &rc) {
 // demonstrate usage of a "pre" fill derived routine
 void PreFill(Container<Real> &rc) {
   MeshBlock *pmb = rc.pmy_block;
-  int is = 0;
-  int js = 0;
-  int ks = 0;
-  int ie = pmb->ncells1 - 1;
-  int je = pmb->ncells2 - 1;
-  int ke = pmb->ncells3 - 1;
+
+  IndexRange ib = pmb->cellbounds.GetBoundsI(IndexDomain::entire);
+  IndexRange jb = pmb->cellbounds.GetBoundsJ(IndexDomain::entire);
+  IndexRange kb = pmb->cellbounds.GetBoundsK(IndexDomain::entire);
+
```

Comment:
Plays nicer with `clang-format` too, which put all the `is,js,ks` calls on separate lines